src/utils/paginationHandler.js

```diff
@@ -7,10 +7,10 @@ const getPagination = (page, size) => {
 
 
 const getPagingData = (data, page, limit) => {
-    const { count: totalItems, rows: stack } = data;
+    const { count: totalItems, rows: result } = data;
```

Comment:
Excellent! you added page limit to the pagination, please keep it up

Reply:
The page Limits are getPagination function and after it sort data result those results in getPagingData 
```suggestion
    const getPagination = (page, size) => {
    const limit = size ? + size : 3;
    const offset = page ? page * limit : 0;
    return { limit, offset };
};
```